### DISCUSSION TITLE
Creating Datasets...

### DISCUSSION THREAD

**Discussion by breitling** [datashim-io/datashim#244] [Q&A]
Noob here...

I am able to create a dataset in my AWS EKS cluster, but it never creates a pvc. Then when I try to create a pod that uses the expected pvc it is always in pending status with an warning about the pvc not found. 

My yaml to create the dataset is typical of the examples I have found in the Datashim docs/material found here.

Is there a way to debug this pvc creation issue? Were should I start?

**Comment by srikumar003** [comment]
@breitling apologies. I don't get notifications for discussions (but I do for issues)

To answer your question, can you paste the output of this command
`$ kubectl describe dataset <your-dataset>`
